Source/controls/plrctrls.cpp

```diff
@@ -1530,6 +1562,40 @@ void PerformPrimaryAction()
 				Slot = jumpSlot;
 				SetCursorPos(mousePos);
 			}
+		} else if (IsStashOpen && GetLeftPanel().Contains(MousePosition)) {
+			Point stashSlot = (ActiveStashSlot != InvalidStashPoint) ? ActiveStashSlot : FindClosestStashSlot(MousePosition);
+
+			// Find any item occupying a slot that is currently under the cursor
+			uint16_t itemUnderCursor = [](Point stashSlot) -> uint16_t {

```

Comment:
nit: Probably makes more sense as a `uint16_t FindStashItemUnderCursor(Point stashSlot)` function rather than a self-evaluating lambda.

Reply:
Being an almost exact copy of the block above it I think I will save this for a later refactor to minimize diff in this PR.